include/deko3d.h

```diff
@@ -1238,6 +1238,7 @@ uint32_t dkMemBlockGetSize(DkMemBlock obj);
 DkResult dkMemBlockFlushCpuCache(DkMemBlock obj, uint32_t offset, uint32_t size);
 
 DkResult dkFenceWait(DkFence* obj, int64_t timeout_ns);
+DkFence dkFenceImport(uint32_t id, uint32_t value);
```

Comment:
Minor nitpick: I think this should be `void dkFenceImport(DkFence* obj, uint32_t id, uint32_t value)` instead. Plus, I'm wondering if it may be worth passing in a pointer to a libnx `NvFence` or `NvMultiFence` object instead of id/value directly.

Reply:
I'm ok with your first suggestion.
As for the second point, the thing is that deko3d.h currently doesn't import the libnx header.

Reply:
I applied your suggestion regarding the function signature